backend/builder/tests/test_task_build_image.py

```diff
@@ -34,7 +34,7 @@ def test_order_building_success(celery_app, celery_worker, mocker, execution_num
     function_2 = orc_mock.FunctionFactory()
 
     # BuildTask `before_start` uses this client to change the status, which would lead to `OrcError`
-    mocker.patch("builder.tasks.task.get_orchestrator_client")
+    mocker.patch("orchestrator.get_orchestrator_client")
```

Comment:
unsure about that change (and the similar one below), I think you usually want to patch as close to the tested function as possible, to avoid nasty side-effects.